### DISCUSSION TITLE
Proposal : add a quick 'ghostty +edit-config` options in the CLI

### DISCUSSION THREAD

**Discussion by pierrelgol** [ghostty-org/ghostty#2899] [Feature Requests, Ideas] [answered]
Hi everyone, I wanted to bring this idea for discussing whether or not people think this would be a useful feature to implement. Basically my idea would be to implement a new command in the ghostty CLI. The command `+edit-config` would use the path of the current config file, and would use `$EDITOR` + path_to_config_file to let you immediately jump into the config file.

Pros : 
- quick access to config file.
- quick edition
- agnostic to editor/platform (although I'm not quite sure how this is done on windows)

Cons : 
- quite niche as I don't believe most people are editing their config files frequently

If there is any issues/discussion on this proposal that already exist, feel free to dismiss it entirely. But in case people find that it would be a good addition I would happily implement it myself. In any case I'd like to hear from others about this proposal. Thanks 😄

**Comment by pluiedev** [comment, marked as answer]
We already have the `open_config` keybind action, though it doesn't use EDITOR (#2961). As suggested in the other thread maybe we should have a separate keybind actions for *editing* a config via the user's editor because we want to reserve `open_config` for opening a GUI config interface (#2354)
